<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11711" started-at="2020-01-14T13:08:00Z" finished-at="2020-01-14T13:08:11Z">
    <groups>
      <group name="Status code">
        <method signature="RestFullWebService.VerifyingContentType()[pri:2, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingContentType" class="apiautomation.RestFullWebService"/>
        <method signature="RestFullWebService.VerifyingServer()[pri:3, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingServer" class="apiautomation.RestFullWebService"/>
        <method signature="RestFullWebService.VerifyingStatusCode()[pri:1, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingStatusCode" class="apiautomation.RestFullWebService"/>
      </group> <!-- Status code -->
      <group name="Save Details">
        <method signature="RestFullWebService.VerifyingVerifyBadRequest()[pri:4, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingVerifyBadRequest" class="apiautomation.RestFullWebService"/>
      </group> <!-- Save Details -->
    </groups>
    <test name="Default test" duration-ms="11711" started-at="2020-01-14T13:08:00Z" finished-at="2020-01-14T13:08:11Z">
      <class name="apiautomation.RestFullWebService">
        <test-method status="PASS" signature="beforeVerifyingStatusCode()[pri:0, instance:apiautomation.RestFullWebService@7d907bac]" name="beforeVerifyingStatusCode" is-config="true" duration-ms="5257" started-at="2020-01-14T15:08:00Z" finished-at="2020-01-14T15:08:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeVerifyingStatusCode -->
        <test-method status="FAIL" signature="VerifyingStatusCode()[pri:1, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingStatusCode" duration-ms="1014" started-at="2020-01-14T15:08:05Z" finished-at="2020-01-14T15:08:06Z">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[expected:<400> but was:<200>]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: expected:<400> but was:<200>
	at junit.framework.Assert.fail(Assert.java:47)
	at junit.framework.Assert.failNotEquals(Assert.java:282)
	at junit.framework.Assert.assertEquals(Assert.java:64)
	at junit.framework.Assert.assertEquals(Assert.java:201)
	at junit.framework.Assert.assertEquals(Assert.java:207)
	at apiautomation.RestFullWebService.VerifyingStatusCode(RestFullWebService.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyingStatusCode -->
        <test-method status="PASS" signature="VerifyingContentType()[pri:2, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingContentType" duration-ms="1030" started-at="2020-01-14T15:08:06Z" finished-at="2020-01-14T15:08:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyingContentType -->
        <test-method status="PASS" signature="VerifyingServer()[pri:3, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingServer" duration-ms="1926" started-at="2020-01-14T15:08:07Z" finished-at="2020-01-14T15:08:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyingServer -->
        <test-method status="PASS" signature="saveDetails()[pri:0, instance:apiautomation.RestFullWebService@7d907bac]" name="saveDetails" is-config="true" duration-ms="1518" started-at="2020-01-14T15:08:09Z" finished-at="2020-01-14T15:08:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- saveDetails -->
        <test-method status="PASS" signature="VerifyingVerifyBadRequest()[pri:4, instance:apiautomation.RestFullWebService@7d907bac]" name="VerifyingVerifyBadRequest" duration-ms="935" started-at="2020-01-14T15:08:10Z" finished-at="2020-01-14T15:08:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyingVerifyBadRequest -->
      </class> <!-- apiautomation.RestFullWebService -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
